backend/src/api/routes/authRoutes.ts

```diff
@@ -18,6 +21,9 @@ router
   .post('/register', run((req: Request) => register(req.body.user)))
   .post('/login', run((req: Request) => login(req.body)))
   .post('/login/google', run((req: Request) => loginWithGoogle(req.body)))
+  .post('/login/facebook', facebookAuthMiddleware, run(
+    (req: Request) => loginWithFacebook(req.user as unknown as IFacebookUser, req.body)
```

Comment:
as unknown as IFacebookUser?

Reply:
i wrote the reason for that type assertion in Slack